### ISSUE TITLE
Error: [next-auth]: `useSession` must be wrapped in a <SessionProvider />

### ISSUE THREAD

**Issue by Abdifatah-said** [adrianhajdin/project_next_13_ai_prompt_sharing#16]
![image](https://github.com/adrianhajdin/project_next_13_ai_prompt_sharing/assets/62906884/a564420f-42f0-4057-a6cf-1f68491a2975)
 
iam encountering this error, how should i fix it. 

thank you.

**Comment by Alxcalex** [comment]
Hi there, did you solved this problem ?

**Comment by Abdifatah-said** [comment]
no still i didnt solve it

**Comment by YugBhanushali** [comment]
In Provider.jsx wrap it with SessionProvider and in layout.js wrap body with Provider

**Comment by Nith567** [comment]
<body>
        <div className='main'>
          <div className='gradient' />
        </div>
        <main className='app'>
        <Provider>
          <Nav/>
          {children}
          </Provider>
        </main>
    </body>

**Comment by Nith567** [comment]
just wrap like this in layout.jsx:
        <Provider>
          <Nav/>
          {children}
          </Provider>

**Comment by Nith567** [comment]
just wrap like this: <body>
        
        <Provider>
          <Nav/>
          {children}
          </Provider>

**Comment by gr8guyrabi** [comment]
check your RootLayout file. it's in /app/layout.jsx

```
 <html lang="en">
        <body>
            <Provider>
                <div className="main">
                    <div className="gradient"></div>
                </div>
                <main className="app">
                    <Nav />
                    {children}
                </main>
            </Provider>
        </body>
    </html>
```

**Comment by yuvraj042003** [comment]
yes If you find this issue then you go to RootLayout and paste this code ` <html lang="en">
        <body>
            <Provider>
                <div className="main">
                    <div className="gradient"></div>
                </div>
                <main className="app">
                    <Nav />
                    {children}
                </main>
            </Provider>
        </body>
    </html>` and resolve this error.

**Comment by yuvraj042003** [comment]
> check your RootLayout component. it's in /app/layout.jsx file
> 
> ```
>  <html lang="en">
>         <body>
>             <Provider>
>                 <div className="main">
>                     <div className="gradient"></div>
>                 </div>
>                 <main className="app">
>                     <Nav />
>                     {children}
>                 </main>
>             </Provider>
>         </body>
>     </html>
> ```

Yes This is right code

**Comment by CreativeRoutine** [comment]
> > check your RootLayout component. it's in /app/layout.jsx file
> > ```
> >  <html lang="en">
> >         <body>
> >             <Provider>
> >                 <div className="main">
> >                     <div className="gradient"></div>
> >                 </div>
> >                 <main className="app">
> >                     <Nav />
> >                     {children}
> >                 </main>
> >             </Provider>
> >         </body>
> >     </html>
> > ```
> 
> Yes This is right code

This renders only <div>Provider</div>. What did i wrong?
